devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/resteasy-example/codestart.yml

```
@@ -2,17 +2,17 @@ name: resteasy-example
 ref: resteasy
 type: code
 tags: example
+metadata:
+  title: RESTEasy JAX-RS example
+  description: Rest is easy peasy with this Hello World RESTEasy resource.
```

```suggestion
  description: REST is easy peasy with this Hello World RESTEasy resource.
```
otherwise it has another meaning :).